Update README.md (#3)
hlrs-tasc/julia-on-hpc-systems: README.md

### Julia depot path
Since the available file systems can differ significantly between HPC centers, it is hard to make a general statement about where the Julia depot folder (by default on

> Unix-like systems: `~/.julia`) should be placed (via [`JULIA_DEPOT_PATH`](https://docs.julialang.org/en/v1/manual/environment-variables/#JULIA_DEPOT_PATH)). Generally speaking, the file system hosting the the Julia depot should have

Reviewer: ```suggestion
Unix-like systems: `~/.julia`) should be placed (via [`JULIA_DEPOT_PATH`](https://docs.julialang.org/en/v1/manual/environment-variables/#JULIA_DEPOT_PATH)).
Generally speaking, the file system hosting the Julia depot should have
```
line length + "the the"

* good (parallel) I/O
* no tight quotas
* read and write access

> * shouldn't be automatically wiped on a regular basis (or should be excluded as an exception)

Reviewer: ```suggestion
* no mechanism for the automatic deletion of unused files (or the depot should be excluded as an exception)
```
to match the leading "[...] the Julia depot should have"

On some systems, it resides in the user's home directory (e.g. at NERSC). On other systems, it is put on a parallel scratch file system (e.g. CSCS and PC2). At the time
of writing (April 2022), there does not seem to be reliable performance data
available that could help to make a data-based decision.

> If multiple platforms, e.g. systems with different architecture, would access the same Julia depot, for example because the fiile system is shared, it might

Reviewer: ```suggestion
If multiple platforms, e.g., systems with different architecture, would access the same Julia depot, for example because the file system is shared, it might
```
